.github/workflows/contracts-build.yml

```diff
@@ -18,6 +18,10 @@ jobs:
         with:
           nix_path: nixpkgs=channel:nixos-unstable
 
+      - name: Run oplint
+        run: |
+          pushd contracts/contracts
+          nix develop .#contracts -c oplint ccip deployable lib mcms
```

Comment:
nit: if you don't need the full `contracts` shell here but just want to run the `oplint` pkg you could just use `nix run .#oplint -- <args>`